docs/src/guide/project-layout.md

```diff
@@ -23,7 +23,79 @@ physic
 This will create the `Proto.toml` file which is the manifest file that buffrs
 uses. The `proto` directory, which is the source directory for all your protocol
 buffer definitions and the `proto/vendor` directory, which contains external
-protocol buffers.
+protocol buffers. Before compilation, a copy of the locally defined definitions
```

Comment:
I would not talk about this, as the vendor folder should not be interesting to the user.